lib/src/signed_video_h26x_common.c

```diff
@@ -962,6 +976,8 @@ hash_and_copy_to_ref(signed_video_t *self, const h26x_nalu_t *nalu, uint8_t *has
     }
     // Copy the |nalu_hash| to |reference_hash| to be used in hash_with_reference().
     memcpy(reference_hash, hash, hash_size);
+    // Copy the |hash| to |linked_hash| while linked hash prinsiple is used.
+    if (self->linked_hash_on) get_linked_hash(self, hash, hash_size);
```

Comment:
When I follow the code with an example of "IPIPIP" stream only one linked hash is stored.